### PR TITLE
add dir hierarchy and tidy up compose up call 

### PR DESCRIPTION
**Pull request by mrjones-plip**
per #873

### CODE REVIEW

File: content/en/apps/tutorials/local-setup.md [hunk 1]
```diff
@@ -55,13 +55,13 @@ Now that you have the dependent tools and software installed, you are ready to s
...
 ```shell
-mkdir -p ~/cht-local-setup/data/couch
+mkdir -p ~/cht-local-setup/couch-data/ && mkdir -p ~/cht-local-setup/core-couch/ && mkdir -p ~/cht-local-setup/upgrade/
 cd ~/cht-local-setup
-curl -s -o ./data/cht-4-core.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic%3Amedic%3Amaster/docker-compose%2Fcht-core.yml && curl -s -o ./data/cht-4-couch.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic%3Amedic%3Amaster/docker-compose%2Fcht-couchdb.yml && curl -s -o cht-upgrader.yml https://raw.githubusercontent.com/medic/cht-upgrade-service/main/docker-compose.yml
-cat > ~/cht-local-setup/env << EOF
-DOCKER_CONFIG_PATH=~/cht-local-setup
-COUCHDB_DATA=~/cht-local-setup/data/couch 
-CHT_COMPOSE_PATH=~/cht-local-setup/data 
+curl -s -o ./core-couch/cht-4-core.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:master/docker-compose/cht-core.yml && curl -s -o ./core-couch/cht-4-couch.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:master/docker-compose/cht-couchdb.yml && curl -s -o ./upgrade/docker-compose.yml https://raw.githubusercontent.com/medic/cht-upgrade-service/main/docker-compose.yml
```
[the remark this answers is not in the document]
Comment (mrjones-plip) [reply content/en/apps/tutorials/local-setup.md]: ah - thank you!  Sorry that was wrong in the first place.  FIxed in b8790b7

**Review by dianabarsan — APPROVED** [review]
Awesome! Thanks for all the changes.

**Comment by andrablaj** [comment]
@mrjones-plip, just a quick update on this change. Esther re-tried the steps after this doc change and we bumped into a path issue while running the `docker-compose up` command. We had to replace the `~` with the absolute path for the `DOCKER_CONFIG_PATH`, `COUCHDB_DATA`  and `CHT_COMPOSE_PATH` to have the container run correctly.

FYI @esthermmoturi

**Comment by mrjones-plip** [comment]
thanks @andrablaj !  See #891

**Pull request merged**